rfcs/rfc1/README.md

```diff
@@ -0,0 +1,211 @@
+
+The Request for Comments (RFC) process is intended to provide a consistent and controlled path for changes to the Open Standards Development specifications and projects maintained by TBD.  The intent is for RFCs to be the primary mechanism for proposing new features, changing governance processes, and documenting design decisions.
+
+Each major decision starts as an RFC.  Everyone is invited to discuss the proposal and work towards a shared understanding of the tradeoffs.  After consensus-building among stakeholders for the RFC, the final decision to accept or reject an RFC is made by the Technical Steering Committee (TSC).  The responsibility of the TSC is to weigh the tradeoffs and to make a decision in light of the overall direction of the relevant specification or project.  Though sometimes arduous, this deliberation is the secret sauce for quality.
+
+This RFC defines the RFC Process itself.
+
+## Motivation
+
+The way TBD develops specifications and SDKs has changed over time.  In the early stages of tbDEX and Web5, development moved quickly, with features added and software published in a fast-paced, exploratory manner.  As we worked towards the 1.0 release, we solidified the feature set and shifted our focus to prioritizing security, ensuring interoperability, and making the SDKs easy to use.
+
+As tbDEX and Web5 continue to mature and grow, it is crucial to establish a more principled and structured approach to introducing changes and new features.  The current decision-making process lacks a well-defined framework, leading to potential inconsistencies and challenges in effectively managing the development lifecycle.  The absence of a clear process for proposing and deciding on major changes can result in a lack of transparency, limited stakeholder involvement, and potential delays in implementation.
+
+To address these challenges, we propose the introduction of a Request for Comments (RFC) process.  The RFC process provides a consistent and controlled path for changes to the Open Standards Development specifications and projects maintained by TBD.  It ensures that major decisions are thoroughly discussed, evaluated, and decided upon in a transparent manner.
+
+The RFC process brings several benefits, including fostering deep, productive discussion and building consensus to make well-informed decisions aligned with the overall direction and goals of the projects.  It also defines an easy-to-follow framework for contributing to the development and implementation of changes, encouraging broader participation.
+
+The introduction of the RFC process marks a significant step forward in the maturity and governance of TBD's development efforts.  By adopting this principled approach, we can ensure the long-term quality, security, and interoperability of our projects while maintaining the agility to adapt to evolving requirements and technological advancements.
+
+## Detailed Design
+
+### <ins>When you need to follow this process</ins>
```

Comment:
what's the reason for `<ins>`? 

one reason to not do `<ins>` is it's not semantically approps and it makes the headings appear clickable

Reply:
just ported this from a doc @frankhinek put together and tried to maintain formatting
usually links are blue, no?

I'm fine moving this to a `####` if that would be better

Reply:
nah links need to be differentiated by something other than colour tecccchnically 

we can keep it as `###` since it comes right after `##`

Reply:
🤔 https://accessibleweb.com/question-answer/do-links-need-to-be-underlined/
fine to keep it as just `###`

Reply:
🧐 https://www.w3.org/TR/WCAG22/#use-of-color